meshers/cubes/voxel_mesher_cubes.cpp

```diff
@@ -343,6 +343,11 @@ void build_voxel_mesh_as_greedy_cubes(
 					Vector3 n;
 					n[za] = m.side == SIDE_FRONT ? -1 : 1;
 
+					Vector3 t = n.cross(v0);
```

Comment:
Why add tangents to this mesher? It only makes coloured cubes. It does not even have UVs.

Reply:
You're not wrong lol. I think this was just the first place I looked and accidentally mentally linked it to the geometry cube. Removed my changes to this mesher.